benchmarks/experiment_runner.py

```diff
@@ -249,8 +251,41 @@ def dump_profile_info(self, prof, model_name):
     with open(os.path.join(file_path, "kernel_dump.txt"), "a") as f:
       f.write(kernel_dump)
 
-  def timed_run(self, benchmark_experiment, benchmark_model):
+  def collect_profile_to_metrics(self, prof, metrics):
+    assert prof is not None, 'Expecting profiler to be defined!'
+    if not self._args.profile_cuda_cpu_collect:
+      logger.warning(
+          'Profiling enabled, but collection of CPU/CUDA profiling info disabled.'
+      )
+      return
+
+    kernel_dump = prof.profiler.total_average()
```

Comment:
Curious where is this total_average() defined?

Reply:
https://github.com/pytorch/pytorch/blob/00412e6dfacdec5a8508c841b3ea0846388dc872/torch/autograd/profiler_util.py#L339-L350